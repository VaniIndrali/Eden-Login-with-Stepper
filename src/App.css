.App {
	margin-top: 3.75rem;
}

h1,
h2,
h3,
h4,
h5,
h6,
input,
button,
label {
	font-family: "Inter";
}

.App-logo {
	height: 2.5rem;
	width: 2.5rem;
}

.workspaceText {
	margin-top: 3.125rem;
	padding: 0;
	padding-bottom: 0.313rem;
	font-size: 1.54rem;
	font-weight: 600;
}

#mainTextEden {
	font-size: 1.59rem;
	padding: 0;
	padding-bottom: 0.313rem;
}

.Companyname {
	margin: 0;
	margin-left: 0.313rem;
	margin-bottom: 2.5rem;
	font-weight: 900;
}

.StepperContainer-0-2-1 {
	align-items: center;
}

.stepperClass {
	border-image: linear-gradient(45deg, indigo 0 50%, white 50% 100%) 1;
}

.StepButton-0-2-4:disabled {
	border: solid;
	border-color: #f2f2f2;
}
