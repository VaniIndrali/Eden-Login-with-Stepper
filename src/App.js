import { React, useState } from "react";
import { Stepper, Step } from "react-form-stepper";
import StepOne from "./StepsComponents/StepsComponentOne/StepOne.js";
import StepTwo from "./StepsComponents/StepsComponentTwo/StepTwo.js";
import StepThree from "./StepsComponents/StepsComponentThree/StepThree.js";
import StepFour from "./StepsComponents/StepsComponentFour/StepFour.js";

import "./App.css";

function App() {
	const [stepValue, setStepValue] = useState(0);
	const [displayName, setDisplayName] = useState("");

	const connectorStyleConfigCss = {
		size: 2,
		stepSize: "1.5em",
	};

	const styleConfigCss = {
		size: "1.9em",
		activeBgColor: "#744ab7",
		inactiveBgColor: "#ffffff",
		inactiveTextColor: "#744ab7",
		completedBgColor: "#744ab7",
	};

	const performStepChange = () => {
		setStepValue(stepValue + 1);
	};

	const getdata = (value) => {
		setDisplayName(value);
		performStepChange();
	};

	return (
		<div className="App">
			<div className="d-flex justify-content-center">
				<img
					src="https://i.imgur.com/CTHLh7p.jpg"
					className="App-logo"
					alt="logo"
				/>
				<h1 className="Companyname">Eden</h1>
			</div>
			<div className="row d-flex justify-content-center">
				<div className="col-md-4">
					<Stepper
						activeStep={stepValue}
						connectorStyleConfig={connectorStyleConfigCss}
						connectorStateColors="true"
						className="stepperClass"
					>
						{/* try this - setStepValue(stepValue+1); */}
						<Step
							onClick={() => setStepValue(0)}
							styleConfig={styleConfigCss}
						/>
						<Step
							onClick={() => setStepValue(1)}
							styleConfig={styleConfigCss}
						/>
						<Step
							onClick={() => setStepValue(2)}
							styleConfig={styleConfigCss}
						/>
						<Step
							onClick={() => setStepValue(3)}
							styleConfig={styleConfigCss}
						/>
					</Stepper>

					{stepValue === 0 && (
						<div className="row d-flex justify-content-center col-md-12">
							<h2 className="text-center mainText">
								Welcome! First things first...
							</h2>
							<StepOne onSubmit={getdata} />
						</div>
					)}
					{stepValue === 1 && (
						<div className="row d-flex justify-content-center col-md-12">
							<h3 className="text-center workspaceText col-12">
								Let's set up a home for all your work
							</h3>
							<StepTwo onSubmit={performStepChange} />
						</div>
					)}
					{stepValue === 2 && (
						<div className="row d-flex justify-content-center col-md-12">
							<h3
								className="text-center mainText col-12"
								id="mainTextEden"
							>
								How are you planning to use Eden?
							</h3>
							<StepThree onSubmit={performStepChange} />
						</div>
					)}
					{stepValue === 3 && <StepFour displayName={displayName} />}
				</div>
			</div>
		</div>
	);
}

export default App;
