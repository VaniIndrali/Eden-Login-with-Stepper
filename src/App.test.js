import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders Element", () => {
	render(<App />);
	const Element = screen.getByText("Eden");
	expect(Element).toBeInTheDocument();
});
