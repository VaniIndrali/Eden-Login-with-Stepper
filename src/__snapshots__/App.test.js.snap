// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`On button clicks should render text accordingly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="d-flex justify-content-center"
        >
          <img
            alt="logo"
            class="App-logo"
            src="https://i.imgur.com/CTHLh7p.jpg"
          />
          <h1
            class="Companyname"
          >
            Eden
          </h1>
        </div>
        <div
          class="row d-flex justify-content-center"
        >
          <div
            class="col-md-4 col-6"
          >
            <div
              class="StepperContainer-0-2-1 stepperClass"
              id="RFS-StepperContainer"
            >
              <div
                class="StepContainer-0-2-2"
                id="RFS-StepContainer"
              >
                <div
                  class="StepMain-0-2-3"
                  id="RFS-StepMain"
                >
                  <button
                    class="StepButton-0-2-4 StepButton-d8-0-2-22 completed"
                    id="RFS-StepButton"
                  >
                    <span
                      class="StepButtonContent-0-2-5 StepButtonContent-d9-0-2-23 completed"
                    >
                      1
                    </span>
                  </button>
                </div>
              </div>
              <div
                class="StepContainer-0-2-2"
                id="RFS-StepContainer"
              >
                <div
                  class="ConnectorContainer-0-2-10 ConnectorContainer-d6-0-2-26"
                  id="RFS-ConnectorContainer"
                >
                  <span
                    class="Connector-0-2-11 Connector-d7-0-2-27 completed"
                    id="RFS-Connector"
                  />
                </div>
                <div
                  class="StepMain-0-2-3"
                  id="RFS-StepMain"
                >
                  <button
                    class="StepButton-0-2-4 StepButton-d10-0-2-24 completed"
                    id="RFS-StepButton"
                  >
                    <span
                      class="StepButtonContent-0-2-5 StepButtonContent-d11-0-2-25 completed"
                    >
                      2
                    </span>
                  </button>
                </div>
              </div>
              <div
                class="StepContainer-0-2-2"
                id="RFS-StepContainer"
              >
                <div
                  class="ConnectorContainer-0-2-10 ConnectorContainer-d8-0-2-30"
                  id="RFS-ConnectorContainer"
                >
                  <span
                    class="Connector-0-2-11 Connector-d9-0-2-31 active"
                    id="RFS-Connector"
                  />
                </div>
                <div
                  class="StepMain-0-2-3"
                  id="RFS-StepMain"
                >
                  <button
                    class="StepButton-0-2-4 StepButton-d12-0-2-28 active"
                    id="RFS-StepButton"
                  >
                    <span
                      class="StepButtonContent-0-2-5 StepButtonContent-d13-0-2-29 active"
                    >
                      3
                    </span>
                  </button>
                </div>
              </div>
              <div
                class="StepContainer-0-2-2"
                id="RFS-StepContainer"
              >
                <div
                  class="ConnectorContainer-0-2-10 ConnectorContainer-d10-0-2-34"
                  id="RFS-ConnectorContainer"
                >
                  <span
                    class="Connector-0-2-11 Connector-d11-0-2-35"
                    id="RFS-Connector"
                  />
                </div>
                <div
                  class="StepMain-0-2-3"
                  id="RFS-StepMain"
                >
                  <button
                    class="StepButton-0-2-4 StepButton-d14-0-2-32"
                    disabled=""
                    id="RFS-StepButton"
                  >
                    <span
                      class="StepButtonContent-0-2-5 StepButtonContent-d15-0-2-33"
                    >
                      4
                    </span>
                  </button>
                </div>
              </div>
            </div>
            <div
              class="row d-flex justify-content-center col-md-12"
            >
              <h3
                class="text-center mainText col-12"
                id="mainTextEden"
              >
                How are you planning to use Eden?
              </h3>
              <div
                id="loginThree"
              >
                <div
                  class="container col-md-12 col-9"
                >
                  <h6
                    class="text-center subText"
                  >
                    We'll streamline your setup experience accordingly.
                  </h6>
                  <div
                    class="row justify-content-center align-items-center"
                    id="login-row"
                  >
                    <div
                      class="col-md-11"
                      id="login-column"
                    >
                      <div
                        class="col-md-12"
                        id="login-box"
                      >
                        <form
                          class="form"
                          id="login-form"
                        >
                          <div
                            class="form-group"
                          >
                            <div
                              class="btn-group"
                            >
                              <button
                                id="btn_myself"
                                type="button"
                              >
                                <svg
                                  class="bi bi-person-fill"
                                  fill="currentColor"
                                  height="27"
                                  id="svgPerson"
                                  viewBox="0 0 16 14"
                                  width="27"
                                  xmlns="http://www.w3.org/2000/svg"
                                >
                                  <path
                                    d="M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H3zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"
                                  />
                                </svg>
                                <h5
                                  class="mt-2"
                                >
                                  For myself
                                </h5>
                                <h6>
                                  Write better. Think more clearly. Stay organized.
                                </h6>
                              </button>
                              <button
                                id="btn_team"
                                type="button"
                              >
                                <img
                                  alt="team"
                                  class="team-logo"
                                  src="https://i.imgur.com/op4AsSO.png"
                                />
                                <h5
                                  class="mt-3"
                                >
                                  With my team
                                </h5>
                                <h6>
                                  Wikis, docs, tasks & projects, all in one place.
                                </h6>
                              </button>
                            </div>
                          </div>
                          <div
                            class="form-group"
                          >
                            <input
                              class="button"
                              name="Create Workspace"
                              type="submit"
                              value="Create Workspace"
                            />
                          </div>
                        </form>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="d-flex justify-content-center"
      >
        <img
          alt="logo"
          class="App-logo"
          src="https://i.imgur.com/CTHLh7p.jpg"
        />
        <h1
          class="Companyname"
        >
          Eden
        </h1>
      </div>
      <div
        class="row d-flex justify-content-center"
      >
        <div
          class="col-md-4 col-6"
        >
          <div
            class="StepperContainer-0-2-1 stepperClass"
            id="RFS-StepperContainer"
          >
            <div
              class="StepContainer-0-2-2"
              id="RFS-StepContainer"
            >
              <div
                class="StepMain-0-2-3"
                id="RFS-StepMain"
              >
                <button
                  class="StepButton-0-2-4 StepButton-d8-0-2-22 completed"
                  id="RFS-StepButton"
                >
                  <span
                    class="StepButtonContent-0-2-5 StepButtonContent-d9-0-2-23 completed"
                  >
                    1
                  </span>
                </button>
              </div>
            </div>
            <div
              class="StepContainer-0-2-2"
              id="RFS-StepContainer"
            >
              <div
                class="ConnectorContainer-0-2-10 ConnectorContainer-d6-0-2-26"
                id="RFS-ConnectorContainer"
              >
                <span
                  class="Connector-0-2-11 Connector-d7-0-2-27 completed"
                  id="RFS-Connector"
                />
              </div>
              <div
                class="StepMain-0-2-3"
                id="RFS-StepMain"
              >
                <button
                  class="StepButton-0-2-4 StepButton-d10-0-2-24 completed"
                  id="RFS-StepButton"
                >
                  <span
                    class="StepButtonContent-0-2-5 StepButtonContent-d11-0-2-25 completed"
                  >
                    2
                  </span>
                </button>
              </div>
            </div>
            <div
              class="StepContainer-0-2-2"
              id="RFS-StepContainer"
            >
              <div
                class="ConnectorContainer-0-2-10 ConnectorContainer-d8-0-2-30"
                id="RFS-ConnectorContainer"
              >
                <span
                  class="Connector-0-2-11 Connector-d9-0-2-31 active"
                  id="RFS-Connector"
                />
              </div>
              <div
                class="StepMain-0-2-3"
                id="RFS-StepMain"
              >
                <button
                  class="StepButton-0-2-4 StepButton-d12-0-2-28 active"
                  id="RFS-StepButton"
                >
                  <span
                    class="StepButtonContent-0-2-5 StepButtonContent-d13-0-2-29 active"
                  >
                    3
                  </span>
                </button>
              </div>
            </div>
            <div
              class="StepContainer-0-2-2"
              id="RFS-StepContainer"
            >
              <div
                class="ConnectorContainer-0-2-10 ConnectorContainer-d10-0-2-34"
                id="RFS-ConnectorContainer"
              >
                <span
                  class="Connector-0-2-11 Connector-d11-0-2-35"
                  id="RFS-Connector"
                />
              </div>
              <div
                class="StepMain-0-2-3"
                id="RFS-StepMain"
              >
                <button
                  class="StepButton-0-2-4 StepButton-d14-0-2-32"
                  disabled=""
                  id="RFS-StepButton"
                >
                  <span
                    class="StepButtonContent-0-2-5 StepButtonContent-d15-0-2-33"
                  >
                    4
                  </span>
                </button>
              </div>
            </div>
          </div>
          <div
            class="row d-flex justify-content-center col-md-12"
          >
            <h3
              class="text-center mainText col-12"
              id="mainTextEden"
            >
              How are you planning to use Eden?
            </h3>
            <div
              id="loginThree"
            >
              <div
                class="container col-md-12 col-9"
              >
                <h6
                  class="text-center subText"
                >
                  We'll streamline your setup experience accordingly.
                </h6>
                <div
                  class="row justify-content-center align-items-center"
                  id="login-row"
                >
                  <div
                    class="col-md-11"
                    id="login-column"
                  >
                    <div
                      class="col-md-12"
                      id="login-box"
                    >
                      <form
                        class="form"
                        id="login-form"
                      >
                        <div
                          class="form-group"
                        >
                          <div
                            class="btn-group"
                          >
                            <button
                              id="btn_myself"
                              type="button"
                            >
                              <svg
                                class="bi bi-person-fill"
                                fill="currentColor"
                                height="27"
                                id="svgPerson"
                                viewBox="0 0 16 14"
                                width="27"
                                xmlns="http://www.w3.org/2000/svg"
                              >
                                <path
                                  d="M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H3zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6z"
                                />
                              </svg>
                              <h5
                                class="mt-2"
                              >
                                For myself
                              </h5>
                              <h6>
                                Write better. Think more clearly. Stay organized.
                              </h6>
                            </button>
                            <button
                              id="btn_team"
                              type="button"
                            >
                              <img
                                alt="team"
                                class="team-logo"
                                src="https://i.imgur.com/op4AsSO.png"
                              />
                              <h5
                                class="mt-3"
                              >
                                With my team
                              </h5>
                              <h6>
                                Wikis, docs, tasks & projects, all in one place.
                              </h6>
                            </button>
                          </div>
                        </div>
                        <div
                          class="form-group"
                        >
                          <input
                            class="button"
                            name="Create Workspace"
                            type="submit"
                            value="Create Workspace"
                          />
                        </div>
                      </form>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`renders App 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="d-flex justify-content-center"
        >
          <img
            alt="logo"
            class="App-logo"
            src="https://i.imgur.com/CTHLh7p.jpg"
          />
          <h1
            class="Companyname"
          >
            Eden
          </h1>
        </div>
        <div
          class="row d-flex justify-content-center"
        >
          <div
            class="col-md-4 col-6"
          >
            <div
              class="StepperContainer-0-2-1 stepperClass"
              id="RFS-StepperContainer"
            >
              <div
                class="StepContainer-0-2-2"
                id="RFS-StepContainer"
              >
                <div
                  class="StepMain-0-2-3"
                  id="RFS-StepMain"
                >
                  <button
                    class="StepButton-0-2-4 StepButton-d0-0-2-6 active"
                    id="RFS-StepButton"
                  >
                    <span
                      class="StepButtonContent-0-2-5 StepButtonContent-d1-0-2-7 active"
                    >
                      1
                    </span>
                  </button>
                </div>
              </div>
              <div
                class="StepContainer-0-2-2"
                id="RFS-StepContainer"
              >
                <div
                  class="ConnectorContainer-0-2-10 ConnectorContainer-d0-0-2-12"
                  id="RFS-ConnectorContainer"
                >
                  <span
                    class="Connector-0-2-11 Connector-d1-0-2-13"
                    id="RFS-Connector"
                  />
                </div>
                <div
                  class="StepMain-0-2-3"
                  id="RFS-StepMain"
                >
                  <button
                    class="StepButton-0-2-4 StepButton-d2-0-2-8"
                    disabled=""
                    id="RFS-StepButton"
                  >
                    <span
                      class="StepButtonContent-0-2-5 StepButtonContent-d3-0-2-9"
                    >
                      2
                    </span>
                  </button>
                </div>
              </div>
              <div
                class="StepContainer-0-2-2"
                id="RFS-StepContainer"
              >
                <div
                  class="ConnectorContainer-0-2-10 ConnectorContainer-d2-0-2-16"
                  id="RFS-ConnectorContainer"
                >
                  <span
                    class="Connector-0-2-11 Connector-d3-0-2-17"
                    id="RFS-Connector"
                  />
                </div>
                <div
                  class="StepMain-0-2-3"
                  id="RFS-StepMain"
                >
                  <button
                    class="StepButton-0-2-4 StepButton-d4-0-2-14"
                    disabled=""
                    id="RFS-StepButton"
                  >
                    <span
                      class="StepButtonContent-0-2-5 StepButtonContent-d5-0-2-15"
                    >
                      3
                    </span>
                  </button>
                </div>
              </div>
              <div
                class="StepContainer-0-2-2"
                id="RFS-StepContainer"
              >
                <div
                  class="ConnectorContainer-0-2-10 ConnectorContainer-d4-0-2-20"
                  id="RFS-ConnectorContainer"
                >
                  <span
                    class="Connector-0-2-11 Connector-d5-0-2-21"
                    id="RFS-Connector"
                  />
                </div>
                <div
                  class="StepMain-0-2-3"
                  id="RFS-StepMain"
                >
                  <button
                    class="StepButton-0-2-4 StepButton-d6-0-2-18"
                    disabled=""
                    id="RFS-StepButton"
                  >
                    <span
                      class="StepButtonContent-0-2-5 StepButtonContent-d7-0-2-19"
                    >
                      4
                    </span>
                  </button>
                </div>
              </div>
            </div>
            <div
              class="row d-flex justify-content-center col-md-12"
            >
              <h2
                class="text-center mainText"
              >
                Welcome! First things first...
              </h2>
              <div
                id="loginOne d-flex justify-content-center"
              >
                <div
                  class="container col-md-12 col-9"
                >
                  <h6
                    class="text-center subText"
                  >
                    You can always change them later.
                  </h6>
                  <div
                    class="row justify-content-center align-items-center"
                    id="login-row"
                  >
                    <div
                      class="col-md-11"
                      id="login-column"
                    >
                      <div
                        class="col-md-12"
                        id="login-box"
                      >
                        <form
                          class="form"
                          id="login-form"
                        >
                          <div
                            class="form-group"
                          >
                            <label
                              class="text-left"
                              for="fullname"
                            >
                              Full Name
                            </label>
                            <br />
                            <input
                              class="form-control"
                              id="fullname"
                              name="fullname"
                              placeholder="Steve Jobs"
                              required=""
                              type="text"
                            />
                          </div>
                          <div
                            class="form-group"
                          >
                            <label
                              class="text-left"
                              for="displayname"
                            >
                              Display Name
                            </label>
                            <br />
                            <input
                              class="form-control"
                              id="displayname"
                              maxlength="11"
                              name="displayname"
                              placeholder="Steve"
                              required=""
                              type="text"
                              value=""
                            />
                          </div>
                          <div
                            class="form-group"
                          >
                            <input
                              class="button"
                              name="Create Workspace"
                              type="submit"
                              value="Create Workspace"
                            />
                          </div>
                        </form>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="d-flex justify-content-center"
      >
        <img
          alt="logo"
          class="App-logo"
          src="https://i.imgur.com/CTHLh7p.jpg"
        />
        <h1
          class="Companyname"
        >
          Eden
        </h1>
      </div>
      <div
        class="row d-flex justify-content-center"
      >
        <div
          class="col-md-4 col-6"
        >
          <div
            class="StepperContainer-0-2-1 stepperClass"
            id="RFS-StepperContainer"
          >
            <div
              class="StepContainer-0-2-2"
              id="RFS-StepContainer"
            >
              <div
                class="StepMain-0-2-3"
                id="RFS-StepMain"
              >
                <button
                  class="StepButton-0-2-4 StepButton-d0-0-2-6 active"
                  id="RFS-StepButton"
                >
                  <span
                    class="StepButtonContent-0-2-5 StepButtonContent-d1-0-2-7 active"
                  >
                    1
                  </span>
                </button>
              </div>
            </div>
            <div
              class="StepContainer-0-2-2"
              id="RFS-StepContainer"
            >
              <div
                class="ConnectorContainer-0-2-10 ConnectorContainer-d0-0-2-12"
                id="RFS-ConnectorContainer"
              >
                <span
                  class="Connector-0-2-11 Connector-d1-0-2-13"
                  id="RFS-Connector"
                />
              </div>
              <div
                class="StepMain-0-2-3"
                id="RFS-StepMain"
              >
                <button
                  class="StepButton-0-2-4 StepButton-d2-0-2-8"
                  disabled=""
                  id="RFS-StepButton"
                >
                  <span
                    class="StepButtonContent-0-2-5 StepButtonContent-d3-0-2-9"
                  >
                    2
                  </span>
                </button>
              </div>
            </div>
            <div
              class="StepContainer-0-2-2"
              id="RFS-StepContainer"
            >
              <div
                class="ConnectorContainer-0-2-10 ConnectorContainer-d2-0-2-16"
                id="RFS-ConnectorContainer"
              >
                <span
                  class="Connector-0-2-11 Connector-d3-0-2-17"
                  id="RFS-Connector"
                />
              </div>
              <div
                class="StepMain-0-2-3"
                id="RFS-StepMain"
              >
                <button
                  class="StepButton-0-2-4 StepButton-d4-0-2-14"
                  disabled=""
                  id="RFS-StepButton"
                >
                  <span
                    class="StepButtonContent-0-2-5 StepButtonContent-d5-0-2-15"
                  >
                    3
                  </span>
                </button>
              </div>
            </div>
            <div
              class="StepContainer-0-2-2"
              id="RFS-StepContainer"
            >
              <div
                class="ConnectorContainer-0-2-10 ConnectorContainer-d4-0-2-20"
                id="RFS-ConnectorContainer"
              >
                <span
                  class="Connector-0-2-11 Connector-d5-0-2-21"
                  id="RFS-Connector"
                />
              </div>
              <div
                class="StepMain-0-2-3"
                id="RFS-StepMain"
              >
                <button
                  class="StepButton-0-2-4 StepButton-d6-0-2-18"
                  disabled=""
                  id="RFS-StepButton"
                >
                  <span
                    class="StepButtonContent-0-2-5 StepButtonContent-d7-0-2-19"
                  >
                    4
                  </span>
                </button>
              </div>
            </div>
          </div>
          <div
            class="row d-flex justify-content-center col-md-12"
          >
            <h2
              class="text-center mainText"
            >
              Welcome! First things first...
            </h2>
            <div
              id="loginOne d-flex justify-content-center"
            >
              <div
                class="container col-md-12 col-9"
              >
                <h6
                  class="text-center subText"
                >
                  You can always change them later.
                </h6>
                <div
                  class="row justify-content-center align-items-center"
                  id="login-row"
                >
                  <div
                    class="col-md-11"
                    id="login-column"
                  >
                    <div
                      class="col-md-12"
                      id="login-box"
                    >
                      <form
                        class="form"
                        id="login-form"
                      >
                        <div
                          class="form-group"
                        >
                          <label
                            class="text-left"
                            for="fullname"
                          >
                            Full Name
                          </label>
                          <br />
                          <input
                            class="form-control"
                            id="fullname"
                            name="fullname"
                            placeholder="Steve Jobs"
                            required=""
                            type="text"
                          />
                        </div>
                        <div
                          class="form-group"
                        >
                          <label
                            class="text-left"
                            for="displayname"
                          >
                            Display Name
                          </label>
                          <br />
                          <input
                            class="form-control"
                            id="displayname"
                            maxlength="11"
                            name="displayname"
                            placeholder="Steve"
                            required=""
                            type="text"
                            value=""
                          />
                        </div>
                        <div
                          class="form-group"
                        >
                          <input
                            class="button"
                            name="Create Workspace"
                            type="submit"
                            value="Create Workspace"
                          />
                        </div>
                      </form>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
